src/lib/operators/join_hash.cpp

```diff
@@ -9,11 +9,9 @@
 #include <utility>
 #include <vector>
 
-#include "bytell_hash_map.hpp"
```

Comment:
Everthing regarding changing the map should be described in the PR + results from benchmarks?